Homework_6/Homework_6/order.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework_6
+{
+    internal class order
```

Comment:
Expected to be a separation between models and services as we described in class - models for data and services for behavior

Reply:
With the folders ofc